src/main/java/com/project/portfolio/springboot/onlinesellingandshopping/serverapp/repository/UserRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.project.portfolio.springboot.onlinesellingandshopping.serverapp.repository;
+
+import com.project.portfolio.springboot.onlinesellingandshopping.serverapp.module.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface UserRepository extends JpaRepository<User,Integer> {
```

Comment:
Missing Javadoc for the method findByUsername. See Effective Java Third Edition Item 56.